scene/2d/sprite.cpp

```diff
@@ -335,10 +335,15 @@ bool Sprite::is_pixel_opaque(const Point2 &p_point) const {
 	if (is_repeat) {
 		int mirror_x = 0;
 		int mirror_y = 0;
+
+		if (texture->get_size().width == 0 || texture->get_size().height == 0)
```

Comment:
I'd move this up after the first `if (texture.is_null())` check. Looking at the code, I don't think it's necessary to let it run through all those calculations to finally return if the input was invalid.